src/SUMMARY.md

```diff
@@ -17,4 +17,5 @@
   - [Communicating with humans](./in-depth/human-communication.md)
   - [Communicating with machines](./in-depth/machine-communication.md)
   - [Rendering documentation for your CLI apps](./in-depth/docs.md)
+  - [Rendering documentation for your Configuration](./in-depth/render-configuration.md)
```

Comment:
What do you feel is the line for this being in the CLI documentation vs being in its own page?